save-frontend/build.gradle.kts

```
@@ -73,6 +73,7 @@ kotlin {
             implementation(npm("os-browserify", "^0.3.0"))
             implementation(npm("path-browserify", "^1.0.1"))
             implementation(npm("react-minimal-pie-chart", "^8.2.0"))
+            implementation(npm("lodash.debounce", "4.0.8"))
```

```suggestion
            implementation(npm("lodash.debounce", "^4.0.8"))
```
Usually it's still preferable to set version ranges and have particular versions fixed in `yarn.lock`